test/test-cid.spec.js

```diff
@@ -704,9 +704,9 @@ describe('CID', () => {
     const cid2 = await new Promise((resolve) => {
       receiver.onmessage = (event) => { resolve(event.data) }
     })
-    assert.strictEqual(cid2.asCID, cid2)
     sender.close()
     receiver.close()
+    assert.strictEqual(cid2['/'], cid2.bytes)
```

Comment:
Moved the assertion to the end so if it fails, we still shut the message channel down and the process doesn't hang forever.